openpype/hosts/max/api/lib.py

```diff
@@ -157,6 +164,63 @@ def get_multipass_setting(project_setting=None):
                            ["multipass"])
 
 
+def set_scene_resolution(width, height):
+    """Set the render resolution
+
+    Args:
+        width(int): value of the width
+        height(int): value of the height
+
+    Returns:
+        None
+
+    """
+    rt.renderWidth = width
+    rt.renderHeight = height
+
+
+def reset_scene_resolution():
+    """Apply the scene resolution from the project definition
+
+    scene resolution can be overwritten by an asset if the asset.data contains
+    any information regarding scene resolution .
+
+    Returns:
+        None
+    """
+    project_name = legacy_io.active_project()
+    project_doc = get_project(project_name)
+    project_data = project_doc["data"]
+    asset_data = get_current_project_asset()["data"]
+
+    # Set project resolution
+    width_key = "resolutionWidth"
+    height_key = "resolutionHeight"
+    proj_width_key = project_data.get(width_key, 1920)
+    proj_height_key = project_data.get(height_key, 1080)
+
+    width = asset_data.get(width_key, proj_width_key)
+    height = asset_data.get(height_key, proj_height_key)
+
+    set_scene_resolution(width, height)
```

Comment:
We should stop using `legacy_io`. Better to use `get_current_project` and `get_current_project_name()` from `openpype.pipeline`:

```suggestion
    project_resolution_data = get_current_project(fields=["data.resolutionWidth", "data.resolutionHeight"])["data"]
    asset_resolution_data = get_current_project_asset(fields=["data.resolutionWidth", "data.resolutionHeight"])["data"]

    # Set project resolution
    project_width = int(project_resolution_data.get("resolutionWidth", 1920))
    project_height = int(project_resolution_data.get("resolutionHeight", 1080))

    width = int(asset_resolution_data .get("resolutionWidth", project_width))
    height = int(asset_resolution_data .get("resolutionHeight", project_height))

    set_scene_resolution(width, height)
```